sourcegraph/sourcegraph
# Tree: consolidate all CSS

## Conext
As we're now [actively working on improving](https://github.com/sourcegraph/sourcegraph/issues/38667) the `Tree` component, this is one of a few small PRs introducing small changes to the code.

## Test plan
1. Check out the branch
2. Ensure tree works as before
3. Make sure tests pass

## App preview:

- [Web](https://sg-web-og-tree-css-consolidate.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-vkfdgnkpig.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
